rtcpeerconnection.go

```diff
@@ -480,9 +480,15 @@ func (pc *RTCPeerConnection) RemoteDescription() *RTCSessionDescription {
 	return pc.CurrentRemoteDescription
 }
 
-// func (pc *RTCPeerConnection) addIceCandidate() {
-// 	panic("not implemented yet") // FIXME NOT-IMPLEMENTED nolint
-// }
+// AddIceCandidate accepts an ICE candidate string and adds it
+// to the existing set of candidates
+func (pc *RTCPeerConnection) AddIceCandidate(s string) {
```

Comment:
Mind updating the signature to `func (pc *RTCPeerConnection) AddIceCandidate(s string) error`

Then instead of printf return an error

Reply:
Ah, yeah. That makes more sense, done.